File: Back-end/Sistem-Rawat-Jalan-RS/app/Models/Pasien.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Pasien extends Model
{
    use HasFactory;
    protected $primaryKey = 'id_pasien';
    protected $fillable = ['nama_pasien', 'no_bpjs', 'nik', 'jenis_kelamin', 'tempat_lahir', 'tanggal_lahir', 'no_hp', 'alamat', 'user_id'];
    public $timestamps = false;

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class, 'user_id', 'id_user');
    }

    public function rekamMedis(): HasMany
    {
        return $this->hasMany(RekamMedis::class, 'pasien_id', 'id_pasien');
    }
}

File: Mobile/Sisrawat/app/src/main/java/com/sisrawat/mobile/data/remote/model/Pasien.kt
package com.sisrawat.mobile.data.remote.model

import com.google.gson.annotations.SerializedName

data class Pasien(
    @field:SerializedName("nama_pasien")
    val namaPasien: String,
    @field:SerializedName("no_bpjs")
    val noBpjs: String,
    @field:SerializedName("nik")
    val nik: String,
    @field:SerializedName("jenis_kelamin")
    val jenisKelamin: String,
    @field:SerializedName("tanggal_lahir")
    val tanggalLahir: String,
    @field:SerializedName("tempat_lahir")
    val tempatLahir: String,
    @field:SerializedName("no_hp")
    val noHp: String,
    @field:SerializedName("alamat")
    val alamat: String
)
